packages/desktop-client/src/components/budget/rollover/CoverMenu.tsx

```diff
@@ -38,7 +56,7 @@ export function CoverMenu({
       <InitialFocus>
         {node => (
           <CategoryAutocomplete
-            categoryGroups={categoryGroups}
+            categoryGroups={removeSelectedCategory(categoryGroups, category)}
```

Comment:
Would this be a good candidate for `useMemo`?

Reply:
I've added this now, it's a _relatively_ expensive calculation.